icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2692,6 +2697,57 @@ class U_I18N_API LocalizedNumberFormatter
 #pragma warning(pop)
 #endif
 
+// TODO: Make it C-compatible
+struct SimpleNumberFormatterOptionsV1 {
+    int32_t minimumFractionDigits = -1;
+    int32_t maximumFractionDigits = -1;
+    int32_t minimumIntegerDigits = -1;
+    int32_t maximumIntegerDigits = -1;
+    int32_t scale = 0;
+};
```

Comment:
The usual way to make a struct forward compatible is to add a first field like `int32_t structLength` that is set to the number of bytes in the struct. Code reading fields should fail with an error if the structLength is too short (even for the v1 struct), and must check the offsetOf(field) for fields added later. Don't put "v1" into the struct name.